common/src/configuration/global.rs

```diff
@@ -794,6 +794,55 @@ fn convert_node_config(
     let metrics = MetricsConfig::from_config(&cfg)?;
     let (base_node_use_libtor, console_wallet_use_libtor) = libtor_enabled(&cfg, net_str);
 
+    let mut node_grpc_server = None;
+    let mut wallet_grpc_server = None;
+    if application == ApplicationType::StratumTranscoder {
+        dbg!("hello from trancoder");
+        let key = "stratum_transcoder.base_node_grpc_address";
+        let pie = cfg.get_str(key);
+        dbg!(&pie);
+        node_grpc_server = Some(
+            pie.map_err(|e| ConfigurationError::new(key, None, &e.to_string()))
+                .and_then(|addr| {
+                    addr.parse::<Multiaddr>()
+                        .map_err(|e| ConfigurationError::new(key, Some(addr), &e.to_string()))
+                })?,
+        );
+        dbg!("hello from walle");
```

Comment:
lol

Reply:
haha there was more than one, I just did not push them.